### PR TITLE
Cleanup all PEP8 problems and migrate to pytest-cov and codecov

### PR DESCRIPTION
**Pull request by Ladsgroup**
[Bug: T179665](https://phabricator.wikimedia.org/T179665)
[Bug: T179668](https://phabricator.wikimedia.org/T179668)

### CODE REVIEW

File: propertysuggester/test/analyzer/test_rule_generator.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
     Entity('Q15', [
         Claim(
             Snak(31, 'wikibase-entityid', 'Q5107'),
-            [Snak(12, 'wikibase-entityid', 'Q123'), Snak(13, 'string', 'qual')],
+            [Snak(12, 'wikibase-entityid', 'Q13'), Snak(13, 'string', 'qual')],
```
Comment (thiemowmde) [inline propertysuggester/test/analyzer/test_rule_generator.py]: This looks like a mistake. Or was there a reason to change this?

File: .travis.yml [hunk 2]
```diff
@@ -6,12 +6,11 @@ python:
 install: 
     - sudo apt-get install libmysqlclient-dev
     - pip install -r requirements.txt
+    - pip install codecov pytest-cov
 
 script:
-    - coverage run --source propertysuggester -m pytest -v
-    - flake8 . --max-line-length=100
```
Comment (thiemowmde) [inline .travis.yml]: I wonder why the line length setting is gone?

File: propertysuggester/test/analyzer/test_rule_generator.py (hunk 1 above)
Comment (Ladsgroup) [reply propertysuggester/test/analyzer/test_rule_generator.py]: It was the funny hack to bring the line length under 80 chars without messing around anything (The value is not important here at all)

File: .travis.yml (hunk 2 above)
Comment (Ladsgroup) [reply .travis.yml]: because the default is 80 (and PEP8 says so too) and I reduced the maximum line to that number so it's not needed anymore.

**Pull request merged**